<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SQL Terminal</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(0, 0, 0);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTextEdit" name="query_entry">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>9</y>
      <width>782</width>
      <height>154</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: green;
background-color: rgb(0, 0, 0);</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="result_text">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>198</y>
      <width>782</width>
      <height>155</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: green;
background-color: rgb(0, 0, 0);</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="exit_button">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>360</y>
      <width>151</width>
      <height>30</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>100</width>
      <height>30</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: red; color: white;</string>
    </property>
    <property name="text">
     <string>Sair - F10</string>
    </property>
   </widget>
   <widget class="QPushButton" name="clear_button">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>360</y>
      <width>151</width>
      <height>30</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>100</width>
      <height>30</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: yellow; color: black;</string>
    </property>
    <property name="text">
     <string>Limpar - F6</string>
    </property>
   </widget>
   <widget class="QPushButton" name="config_button">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>360</y>
      <width>151</width>
      <height>30</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>100</width>
      <height>30</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: blue; color: white;</string>
    </property>
    <property name="text">
     <string>Configuração - F7</string>
    </property>
   </widget>
   <widget class="QPushButton" name="execute_button">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>360</y>
      <width>151</width>
      <height>30</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>100</width>
      <height>30</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: green; color: white;</string>
    </property>
    <property name="text">
     <string>Executar - F5</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
